org.mixedrealitytoolkit.input/Subsystems/Hands/XRSDKHandsSubsystem.cs

```diff
@@ -22,7 +21,7 @@ namespace MixedReality.Toolkit.Input
         Name = "org.mixedrealitytoolkit.xrsdkhands",
         DisplayName = "Subsystem for XRSDK Hands API",
         Author = "Mixed Reality Toolkit Contributors",
-        ProviderType = typeof(XRSDKProvider),
+        ProviderType = typeof(HandsProvider<XRSDKHandContainer>),
```

Comment:
Similar question re: XRSDKHandContainer

Reply:
They're existing private classes defined in their corresponding subsystem files.

https://github.com/MixedRealityToolkit/MixedRealityToolkit-Unity/blob/62dca36d8fdae7c0c17910f10b1eca2f22ed3e70/org.mixedrealitytoolkit.input/Subsystems/Hands/XRSDKHandsSubsystem.cs#L49